### PR TITLE
feat: Collators table

### LINKED ISSUE #62: Add pagination to Collator's list
Collators table should include pagination, as shown in the designs:

Screenshot from 2023-02-28 10-08-31

### Technical considerations
- There's a related PR that includes a paginated Table (see #64).
- Replace the `<table>` component in `Collators.tsx` with  the paginated component

### PR DESCRIPTION
**Pull request by nejcm**
Closes #62

### CODE REVIEW

File: src/pages/collators/Collators.tsx [hunk 1]
```diff
@@ -50,133 +42,76 @@ export function Collators() {
...
           collator: candidate.id,
           totalStaked: format(totalStaked, tokenSymbol),
           delegators: candidate.delegators.length,
-          apy: inflationInfo?.delegator.rewardRate.annual || "0.00%",
+          apy: inflationInfo?.delegator.rewardRate.annual || '0.00%',
         };
-        return rowItem;
       }),
-    [candidates, inflationInfo?.delegator.rewardRate.annual, tokenSymbol]
+    [candidates, inflationInfo?.delegator.rewardRate.annual, tokenSymbol],
```
Comment (gonzamontiel) [inline src/pages/collators/Collators.tsx:97]: This extra comma is from the lint? is it necessary?

**Review by gonzamontiel — COMMENTED** [review]
As we spoke, the idea of the pagination was to avoid the overflow that occurs here.
An easy fix (instead of calculating rows height and viewport height), would be to calculate, in average the amount of rows that should fit in the viewport in a Desktop, and that will work in most cases. 
Also, didn't find the culprit, but the table border shouldn't be that clear, but like the inner rows border.

Screenshot from 2023-03-14 14 49 06

**Comment by nejcm** [comment]
@gonzamontiel Updated

**Review by gonzamontiel — APPROVED** [review]
I've added some changes and I think it is ready to merge.
The advantage of having an abstract Table with pagination (thanks @nejcm) comes with an inevitable loss in code readability.
We don't have the time now to make it perfect, but I think there is room for improvement.
We should keep in mind for future PRs

**Pull request merged**